Makefile

```diff
@@ -80,17 +79,17 @@ setup-git: ensure-venv setup-git-config
 	@echo ""
 
 node-version-check:
-	@test "$$(node -v)" = v"$$(cat .nvmrc)" || (echo 'node version does not match .nvmrc. Recommended to use https://github.com/volta-cli/volta'; exit 1)
+	@test "$$(node -v)" = v10.16.3 || (echo 'node version does not match 10.16.3. Recommended to use https://github.com/volta-cli/volta'; exit 1)
```

Comment:
Blocker: I think this test should not exist or if it does, we should be reading this value from `package.json` file.

Reply:
Yeah, unfortunately, I wouldn't want to have people install `jq`. Would you be fine with a shitty `grep` for this?

My opinion is that it's not worth the effort, given that we rarely change node versions. When we do, it'd be easy to change this value. (It'd error pretty quickly in CI anyways.)

Reply:
>Yeah, unfortunately, I wouldn't want to have people install jq. Would you be fine with a shitty grep for this?

Yes, I'd be okay with a shitty grep. I'll even save you from the hassle:
```bash
awk 'BEGIN { RS=",|: {\n"; FS="\""; } $2 == "node" { print $4 }' package.json
```

>My opinion is that it's not worth the effort, given that we rarely change node versions. When we do, it'd be easy to change this value. (It'd error pretty quickly in CI anyways.)

Trust me, when we need to change the node version, and we will soon as [they are releasing a new security update soon](https://nodejs.org/en/blog/vulnerability/june-2020-security-releases/), this will become a problem. Even worse, 6 months later, when someone who is not you (or even you) wants to update this, it will again be a big headache.

Reply:
I just don't see why it would be a "big headache" to update one thing that is even caught by CI.

Thanks for the awk, I'll put it in. Just to clarify, I don't have a problem with doing this at all, I'm just interested in why you're so averse to this.

Reply:
Oh, your awk doesn't work.

Reply:
>Oh, your awk doesn't work.

It works on my machine 😛 Joking aside, I think this is one of those shitty BSD/GNU/whatever differences as I tested this on Ubuntu and guessing you are trying this on macOS.

Reply:
>Thanks for the awk, I'll put it in. Just to clarify, I don't have a problem with doing this at all, I'm just interested in why you're so averse to this.

For anyone reading this who is not me or Josh, we discussed this over Slack. Summary: because it is not obvious or easy to find/remember that this needs be changed until it fails.

Reply:
For posterity it's GNU awk vs BSD awk. I can come up with a fix.

Reply:
Oh, comments just refreshed.